frontend/src/component/project/Project/ProjectDoraMetrics/ProjectDoraFeedback/ProjectDoraFeedback.tsx

```diff
@@ -0,0 +1,158 @@
+import { useState, useEffect } from 'react';
+import { Box, Button, Divider, Typography, styled } from '@mui/material';
+import { ChatBubble, PermMedia, Send } from '@mui/icons-material';
+import { usePlausibleTracker } from 'hooks/usePlausibleTracker';
+import { createLocalStorage } from 'utils/createLocalStorage';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+
+const StyledOuterContainer = styled(Box)(({ theme }) => ({
+    backgroundColor: theme.palette.background.paper,
+    padding: theme.spacing(2, 4),
+    borderRadius: theme.shape.borderRadiusLarge,
+    marginBottom: theme.spacing(2),
+}));
+
+const StyledBtnContainer = styled(Box)(({ theme }) => ({
+    marginTop: theme.spacing(3),
+}));
+
+const StyledDivider = styled(Divider)(({ theme }) => ({
+    marginTop: theme.spacing(3),
+    marginBottom: theme.spacing(1.5),
+}));
+
+const StyledFlexBox = styled(Box)(({ theme }) => ({
+    display: 'flex',
+    alignItems: 'center',
+    marginTop: theme.spacing(1),
+    marginRight: theme.spacing(3),
+}));
+
+const StyledIconWrapper = styled(Box)(({ theme }) => ({
+    color: theme.palette.primary.main,
+    marginRight: theme.spacing(1),
+    display: 'flex',
+    alignItems: 'center',
+}));
+
+const StyledHeader = styled(Typography)(({ theme }) => ({
+    fontSize: theme.fontSizes.mainHeader,
+    marginBottom: theme.spacing(2),
+    fontWeight: 'bold',
+}));
+
+const StyledLink = styled('a')(({ theme }) => ({
+    textDecoration: 'none',
+}));
+
+export const ProjectDoraFeedback = () => {
+    const { trackEvent } = usePlausibleTracker();
+    const { value, setValue } = createLocalStorage(
+        `project:metrics:plausible`,
+        { sent: false }
+    );
+    const [sentObj, setSentObj] = useState(value);
```

Comment:
Can this have more meaningful name?

Reply:
It could be `sent`, but then it would be `sent.sent` which is not ideal. I will think on it.

Reply:
What are we sending, metrics? Maybe it can be called something like that. `Sent` is already in payload.

Reply:
Much better. Thank you.